package.json

```diff
@@ -9,6 +9,7 @@
     "serve": "vue-cli-service serve",
     "serve-backend": "npx nodemon ./backend/index.js",
     "start": "node ./backend/index.js",
+    "build": "vue-cli-service build",
     "prestart": "vue-cli-service build",
```

Comment:
Would be better to change `prestart` to `npm run build`, so if we change build, we don't have to change in two places.